satpy/composites/spectral.py

```diff
@@ -114,8 +114,8 @@ class NDVIHybridGreen(SpectralBlender):
 
     This green band correction follows the same approach as the HybridGreen compositor, but with a dynamic blend
     factor `f` that depends on the pixel-level Normalized Differece Vegetation Index (NDVI). The higher the NDVI, the
-    smaller the contribution from the nir channel will be, following a liner relationship between the two ranges
-    `[ndvi_min, ndvi_max]` and `limits`.
+    smaller the contribution from the nir channel will be, following a liner (default) or non-linear relationship
+    between the two ranges `[ndvi_min, ndvi_max]` and `limits`.
```

Comment:
This is from another PR, right?

Reply:
hmm yes.. Seems like I may have branched out from the branch used for the other PR.. May I ask what's the best way to deal with this? Reset this branch or leave it as if for now and then pull in master and modify accordingly once the other PR has been merged?

Reply:
I don't know if it's the right way, but I'd just remove here the code that is implemented in the other PR, and commit that. And then later merge the main branch.